Python 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 16:02:32) [MSC v.1900 64 bit (AMD64)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> 
============== RESTART: C:/Users/Administrator/Desktop/conv.py ==============
Traceback (most recent call last):
  File "C:/Users/Administrator/Desktop/conv.py", line 1, in <module>
    import matplotlib.pyplot as plt
ImportError: No module named 'matplotlib'
>>> 
============== RESTART: C:/Users/Administrator/Desktop/conv.py ==============
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:/Users/Administrator/Desktop/conv.py", line 1, in <module>
    from tensorflow.examples.tutorials.mnist import input_data
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 60, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'


Error importing tensorflow.  Unless you are using bazel,
you should not try to import tensorflow from its source directory;
please exit the tensorflow source tree, and relaunch your python interpreter
from there.
>>> 
============== RESTART: C:/Users/Administrator/Desktop/conv.py ==============
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:/Users/Administrator/Desktop/conv.py", line 1, in <module>
    from tensorflow.examples.tutorials.mnist import input_data
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 60, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'


Error importing tensorflow.  Unless you are using bazel,
you should not try to import tensorflow from its source directory;
please exit the tensorflow source tree, and relaunch your python interpreter
from there.
>>> from tensorflow.examples.tutorials.mnist import input_data
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<pyshell#0>", line 1, in <module>
    from tensorflow.examples.tutorials.mnist import input_data
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 60, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'


Error importing tensorflow.  Unless you are using bazel,
you should not try to import tensorflow from its source directory;
please exit the tensorflow source tree, and relaunch your python interpreter
from there.
>>> File "<frozen importlib._bootstrap>", line 577, in module_from_spec
SyntaxError: invalid syntax
>>> import tensorflow as tf
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<pyshell#2>", line 1, in <module>
    import tensorflow as tf
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 60, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'


Error importing tensorflow.  Unless you are using bazel,
you should not try to import tensorflow from its source directory;
please exit the tensorflow source tree, and relaunch your python interpreter
from there.
>>> 
============== RESTART: C:\Users\Administrator\Desktop\test.py ==============
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\test.py", line 1, in <module>
    from tensorflow.examples.tutorials.mnist import input_data
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 60, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 914, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: DLL load failed: 找不到指定的模块。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\__init__.py", line 54, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 21, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 20, in swig_import_helper
    return importlib.import_module('_pywrap_tensorflow')
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_pywrap_tensorflow'


Error importing tensorflow.  Unless you are using bazel,
you should not try to import tensorflow from its source directory;
please exit the tensorflow source tree, and relaunch your python interpreter
from there.
>>> 
============== RESTART: C:\Users\Administrator\Desktop\test.py ==============
Successfully downloaded train-images-idx3-ubyte.gz 9912422 bytes.
Extracting MNIST_data/train-images-idx3-ubyte.gz
Successfully downloaded train-labels-idx1-ubyte.gz 28881 bytes.
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Successfully downloaded t10k-images-idx3-ubyte.gz 1648877 bytes.
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Successfully downloaded t10k-labels-idx1-ubyte.gz 4542 bytes.
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.1
step 1, training accuracy 0.14
step 2, training accuracy 0.1
step 3, training accuracy 0.2
step 4, training accuracy 0.18
step 5, training accuracy 0.14
step 6, training accuracy 0.22
step 7, training accuracy 0.24
step 8, training accuracy 0.24
step 9, training accuracy 0.18
step 10, training accuracy 0.26
step 11, training accuracy 0.32
step 12, training accuracy 0.26
step 13, training accuracy 0.4
step 14, training accuracy 0.4
step 15, training accuracy 0.38
step 16, training accuracy 0.48
step 17, training accuracy 0.46
step 18, training accuracy 0.52
step 19, training accuracy 0.44
step 20, training accuracy 0.48
step 21, training accuracy 0.68
step 22, training accuracy 0.64
step 23, training accuracy 0.56
step 24, training accuracy 0.68
step 25, training accuracy 0.58
step 26, training accuracy 0.6
step 27, training accuracy 0.6
step 28, training accuracy 0.66
step 29, training accuracy 0.5
step 30, training accuracy 0.72
step 31, training accuracy 0.74
step 32, training accuracy 0.84
step 33, training accuracy 0.64
step 34, training accuracy 0.72
step 35, training accuracy 0.68
step 36, training accuracy 0.54
step 37, training accuracy 0.7
step 38, training accuracy 0.76
step 39, training accuracy 0.7
step 40, training accuracy 0.58
step 41, training accuracy 0.74
step 42, training accuracy 0.86
step 43, training accuracy 0.66
step 44, training accuracy 0.54
step 45, training accuracy 0.56
step 46, training accuracy 0.66
step 47, training accuracy 0.66
step 48, training accuracy 0.72
step 49, training accuracy 0.7
step 50, training accuracy 0.68
step 51, training accuracy 0.84
step 52, training accuracy 0.7
step 53, training accuracy 0.76
step 54, training accuracy 0.74
step 55, training accuracy 0.7
step 56, training accuracy 0.86
step 57, training accuracy 0.72
step 58, training accuracy 0.76
step 59, training accuracy 0.76
step 60, training accuracy 0.66
step 61, training accuracy 0.88
step 62, training accuracy 0.74
step 63, training accuracy 0.88
step 64, training accuracy 0.76
step 65, training accuracy 0.88
step 66, training accuracy 0.88
step 67, training accuracy 0.86
step 68, training accuracy 0.78
step 69, training accuracy 0.84
step 70, training accuracy 0.84
step 71, training accuracy 0.88
step 72, training accuracy 0.82
step 73, training accuracy 0.74
step 74, training accuracy 0.72
step 75, training accuracy 0.72
step 76, training accuracy 0.86
step 77, training accuracy 0.62
step 78, training accuracy 0.84
step 79, training accuracy 0.82
step 80, training accuracy 0.92
step 81, training accuracy 0.82
step 82, training accuracy 0.8
step 83, training accuracy 0.8
step 84, training accuracy 0.82
step 85, training accuracy 0.8
step 86, training accuracy 0.84
step 87, training accuracy 0.84
step 88, training accuracy 0.82
step 89, training accuracy 0.88
step 90, training accuracy 0.84
step 91, training accuracy 0.82
step 92, training accuracy 0.88
step 93, training accuracy 0.84
step 94, training accuracy 0.9
step 95, training accuracy 0.8
step 96, training accuracy 0.92
step 97, training accuracy 0.9
step 98, training accuracy 0.94
step 99, training accuracy 0.9
>>> 
============== RESTART: C:\Users\Administrator\Desktop\test.py ==============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.08
step 1, training accuracy 0.08
step 2, training accuracy 0.1
step 3, training accuracy 0.08
step 4, training accuracy 0.12
step 5, training accuracy 0.12
step 6, training accuracy 0.24
step 7, training accuracy 0.2
step 8, training accuracy 0.06
step 9, training accuracy 0.26
step 10, training accuracy 0.28
step 11, training accuracy 0.36
step 12, training accuracy 0.28
step 13, training accuracy 0.48
step 14, training accuracy 0.2
step 15, training accuracy 0.38
step 16, training accuracy 0.34
step 17, training accuracy 0.38
step 18, training accuracy 0.48
step 19, training accuracy 0.48
step 20, training accuracy 0.4
step 21, training accuracy 0.38
step 22, training accuracy 0.66
step 23, training accuracy 0.48
step 24, training accuracy 0.52
step 25, training accuracy 0.42
step 26, training accuracy 0.42
step 27, training accuracy 0.6
step 28, training accuracy 0.6
step 29, training accuracy 0.62
step 30, training accuracy 0.6
step 31, training accuracy 0.78
step 32, training accuracy 0.72
step 33, training accuracy 0.58
step 34, training accuracy 0.66
step 35, training accuracy 0.68
step 36, training accuracy 0.56
step 37, training accuracy 0.64
step 38, training accuracy 0.66
step 39, training accuracy 0.62
step 40, training accuracy 0.72
step 41, training accuracy 0.66
step 42, training accuracy 0.7
step 43, training accuracy 0.64
step 44, training accuracy 0.54
step 45, training accuracy 0.66
step 46, training accuracy 0.78
step 47, training accuracy 0.62
step 48, training accuracy 0.74
step 49, training accuracy 0.68
step 50, training accuracy 0.72
step 51, training accuracy 0.74
step 52, training accuracy 0.64
step 53, training accuracy 0.74
step 54, training accuracy 0.76
step 55, training accuracy 0.68
step 56, training accuracy 0.72
step 57, training accuracy 0.74
step 58, training accuracy 0.72
step 59, training accuracy 0.76
step 60, training accuracy 0.74
step 61, training accuracy 0.86
step 62, training accuracy 0.58
step 63, training accuracy 0.88
step 64, training accuracy 0.8
step 65, training accuracy 0.78
step 66, training accuracy 0.78
step 67, training accuracy 0.9
step 68, training accuracy 0.74
step 69, training accuracy 0.86
step 70, training accuracy 0.88
step 71, training accuracy 0.88
step 72, training accuracy 0.78
step 73, training accuracy 0.76
step 74, training accuracy 0.74
step 75, training accuracy 0.78
step 76, training accuracy 0.88
step 77, training accuracy 0.58
step 78, training accuracy 0.84
step 79, training accuracy 0.82
step 80, training accuracy 0.94
step 81, training accuracy 0.94
step 82, training accuracy 0.84
step 83, training accuracy 0.88
step 84, training accuracy 0.78
step 85, training accuracy 0.84
step 86, training accuracy 0.8
step 87, training accuracy 0.76
step 88, training accuracy 0.8
step 89, training accuracy 0.84
step 90, training accuracy 0.9
step 91, training accuracy 0.78
step 92, training accuracy 0.9
step 93, training accuracy 0.82
step 94, training accuracy 0.78
step 95, training accuracy 0.78
step 96, training accuracy 0.88
step 97, training accuracy 0.92
step 98, training accuracy 0.9
step 99, training accuracy 0.9
>>> 
============== RESTART: C:\Users\Administrator\Desktop\test.py ==============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.12
step 100, training accuracy 0.82
step 200, training accuracy 0.92
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\test.py", line 67, in <module>
    train_step.run(feed_dict={x: batch[0], y_: batch[1], keep_prob: 0.5})
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 1449, in run
    _run_using_default_session(self, feed_dict, self.graph, session)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 3668, in _run_using_default_session
    session.run(operation, feed_dict)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 766, in run
    run_metadata_ptr)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 964, in _run
    feed_dict_string, options, run_metadata)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1014, in _do_run
    target_list, options, run_metadata)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1021, in _do_call
    return fn(*args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1003, in _run_fn
    status, run_metadata)
KeyboardInterrupt
>>> 
============== RESTART: C:\Users\Administrator\Desktop\test.py ==============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.12
step 50, training accuracy 0.68
step 100, training accuracy 0.76
step 150, training accuracy 0.8
step 200, training accuracy 0.94
step 250, training accuracy 0.88
step 300, training accuracy 0.86
step 350, training accuracy 0.78
step 400, training accuracy 0.94
step 450, training accuracy 0.92
step 500, training accuracy 0.96
step 550, training accuracy 0.96
step 600, training accuracy 1
step 650, training accuracy 0.88
step 700, training accuracy 0.96
step 750, training accuracy 0.86
step 800, training accuracy 0.9
step 850, training accuracy 1
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\test.py", line 67, in <module>
    train_step.run(feed_dict={x: batch[0], y_: batch[1], keep_prob: 0.5})
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 1449, in run
    _run_using_default_session(self, feed_dict, self.graph, session)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 3668, in _run_using_default_session
    session.run(operation, feed_dict)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 766, in run
    run_metadata_ptr)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 964, in _run
    feed_dict_string, options, run_metadata)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1014, in _do_run
    target_list, options, run_metadata)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1021, in _do_call
    return fn(*args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1003, in _run_fn
    status, run_metadata)
KeyboardInterrupt
>>> 
============== RESTART: C:\Users\Administrator\Desktop\test.py ==============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.1
step 50, training accuracy 0.7
step 100, training accuracy 0.8
step 150, training accuracy 0.82
step 200, training accuracy 0.92
step 250, training accuracy 0.9
step 300, training accuracy 0.86
step 350, training accuracy 0.82
step 400, training accuracy 1
step 450, training accuracy 0.94
step 500, training accuracy 0.92
step 550, training accuracy 0.98
step 600, training accuracy 1
step 650, training accuracy 0.92
step 700, training accuracy 0.98
step 750, training accuracy 0.9
step 800, training accuracy 0.84
step 850, training accuracy 1
step 900, training accuracy 1
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\test.py", line 67, in <module>
    train_step.run(feed_dict={x: batch[0], y_: batch[1], keep_prob: 0.5})
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 1449, in run
    _run_using_default_session(self, feed_dict, self.graph, session)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 3668, in _run_using_default_session
    session.run(operation, feed_dict)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 766, in run
    run_metadata_ptr)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 964, in _run
    feed_dict_string, options, run_metadata)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1014, in _do_run
    target_list, options, run_metadata)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1021, in _do_call
    return fn(*args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\client\session.py", line 1003, in _run_fn
    status, run_metadata)
KeyboardInterrupt
>>> 
============== RESTART: C:\Users\Administrator\Desktop\test.py ==============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.14
step 50, training accuracy 0.68
step 100, training accuracy 0.82
step 150, training accuracy 0.8
step 200, training accuracy 0.92
step 250, training accuracy 0.84
step 300, training accuracy 0.88
step 350, training accuracy 0.7
step 400, training accuracy 0.98
step 450, training accuracy 0.9
step 500, training accuracy 0.92
step 550, training accuracy 0.98
step 600, training accuracy 1
step 650, training accuracy 0.9
step 700, training accuracy 0.96
step 750, training accuracy 0.88
step 800, training accuracy 0.88
step 850, training accuracy 1
step 900, training accuracy 1
step 950, training accuracy 0.98
step 1000, training accuracy 0.94
step 1050, training accuracy 0.94
step 1100, training accuracy 0.96
step 1150, training accuracy 0.98
step 1200, training accuracy 0.9
step 1250, training accuracy 0.98
step 1300, training accuracy 1
step 1350, training accuracy 0.9
step 1400, training accuracy 0.96
step 1450, training accuracy 0.96
step 1500, training accuracy 0.94
step 1550, training accuracy 0.96
step 1600, training accuracy 0.96
step 1650, training accuracy 0.98
step 1700, training accuracy 0.94
step 1750, training accuracy 0.98
step 1800, training accuracy 1
step 1850, training accuracy 0.98
step 1900, training accuracy 0.96
step 1950, training accuracy 0.96
step 2000, training accuracy 1
step 2050, training accuracy 0.96
step 2100, training accuracy 0.98
step 2150, training accuracy 0.96
step 2200, training accuracy 0.98
step 2250, training accuracy 1
step 2300, training accuracy 1
step 2350, training accuracy 1
step 2400, training accuracy 1
step 2450, training accuracy 1
step 2500, training accuracy 1
step 2550, training accuracy 1
step 2600, training accuracy 0.98
step 2650, training accuracy 0.98
step 2700, training accuracy 0.98
step 2750, training accuracy 0.94
step 2800, training accuracy 0.94
step 2850, training accuracy 0.98
step 2900, training accuracy 0.98
step 2950, training accuracy 0.98
step 3000, training accuracy 1
step 3050, training accuracy 1
step 3100, training accuracy 0.98
step 3150, training accuracy 0.98
step 3200, training accuracy 0.98
step 3250, training accuracy 0.98
step 3300, training accuracy 1
step 3350, training accuracy 0.98
step 3400, training accuracy 0.98
step 3450, training accuracy 0.98
step 3500, training accuracy 1
step 3550, training accuracy 1
step 3600, training accuracy 0.98
step 3650, training accuracy 0.94
step 3700, training accuracy 0.98
step 3750, training accuracy 1
step 3800, training accuracy 1
step 3850, training accuracy 0.98
step 3900, training accuracy 1
step 3950, training accuracy 0.98
step 4000, training accuracy 1
step 4050, training accuracy 1
step 4100, training accuracy 0.98
step 4150, training accuracy 1
step 4200, training accuracy 1
step 4250, training accuracy 1
step 4300, training accuracy 0.98
step 4350, training accuracy 0.98
step 4400, training accuracy 1
step 4450, training accuracy 1
step 4500, training accuracy 0.98
step 4550, training accuracy 0.98
step 4600, training accuracy 0.98
step 4650, training accuracy 1
step 4700, training accuracy 1
step 4750, training accuracy 1
step 4800, training accuracy 1
step 4850, training accuracy 0.98
step 4900, training accuracy 0.98
step 4950, training accuracy 1
test accuracy 0.9878
>>> 
============ RESTART: C:\Users\Administrator\Desktop\conv\test.py ============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.07
test accuracy 0.0548
Model saved in file: ./digit
>>> 
============ RESTART: C:\Users\Administrator\Desktop\conv\test.py ============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.06
test accuracy 0.2702
Model saved in file: ./digit
>>> 
============ RESTART: C:\Users\Administrator\Desktop\conv\test.py ============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.06
step 50, training accuracy 0.76
step 100, training accuracy 0.91
test accuracy 0.8931
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\conv\test.py", line 74, in <module>
    print("Model saved in file: %s" % save_path)
NameError: name 'save_path' is not defined
>>> 
============ RESTART: C:\Users\Administrator\Desktop\conv\test.py ============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.11
step 50, training accuracy 0.77
step 100, training accuracy 0.93
test accuracy 0.8975
>>> 
============ RESTART: C:\Users\Administrator\Desktop\conv\test.py ============
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
step 0, training accuracy 0.07
step 50, training accuracy 0.76
step 100, training accuracy 0.89
step 150, training accuracy 0.84
step 200, training accuracy 0.94
step 250, training accuracy 0.9
step 300, training accuracy 0.96
step 350, training accuracy 0.95
step 400, training accuracy 0.88
step 450, training accuracy 0.94
step 500, training accuracy 0.96
step 550, training accuracy 0.92
step 600, training accuracy 0.92
step 650, training accuracy 0.96
step 700, training accuracy 1
step 750, training accuracy 0.96
step 800, training accuracy 0.96
step 850, training accuracy 0.96
step 900, training accuracy 0.95
step 950, training accuracy 0.94
step 1000, training accuracy 0.99
step 1050, training accuracy 0.97
step 1100, training accuracy 0.97
step 1150, training accuracy 0.97
step 1200, training accuracy 0.95
step 1250, training accuracy 0.99
step 1300, training accuracy 0.94
step 1350, training accuracy 0.98
step 1400, training accuracy 0.97
step 1450, training accuracy 0.96
step 1500, training accuracy 1
step 1550, training accuracy 0.98
step 1600, training accuracy 0.98
step 1650, training accuracy 0.98
step 1700, training accuracy 1
step 1750, training accuracy 0.98
step 1800, training accuracy 0.99
step 1850, training accuracy 0.96
step 1900, training accuracy 0.99
step 1950, training accuracy 0.99
step 2000, training accuracy 0.97
step 2050, training accuracy 0.99
step 2100, training accuracy 0.99
step 2150, training accuracy 0.98
step 2200, training accuracy 1
step 2250, training accuracy 0.99
step 2300, training accuracy 0.96
step 2350, training accuracy 0.98
step 2400, training accuracy 0.99
step 2450, training accuracy 0.98
step 2500, training accuracy 0.99
step 2550, training accuracy 0.98
step 2600, training accuracy 0.99
step 2650, training accuracy 1
step 2700, training accuracy 0.98
step 2750, training accuracy 1
step 2800, training accuracy 0.99
step 2850, training accuracy 0.99
step 2900, training accuracy 0.99
step 2950, training accuracy 0.99
step 3000, training accuracy 0.99
step 3050, training accuracy 0.99
step 3100, training accuracy 0.98
step 3150, training accuracy 0.99
step 3200, training accuracy 0.98
step 3250, training accuracy 0.98
step 3300, training accuracy 0.99
step 3350, training accuracy 0.98
step 3400, training accuracy 0.98
step 3450, training accuracy 1
step 3500, training accuracy 1
step 3550, training accuracy 0.99
step 3600, training accuracy 0.97
step 3650, training accuracy 1
step 3700, training accuracy 1
step 3750, training accuracy 1
step 3800, training accuracy 0.99
step 3850, training accuracy 1
step 3900, training accuracy 1
step 3950, training accuracy 1
step 4000, training accuracy 0.99
step 4050, training accuracy 1
step 4100, training accuracy 1
step 4150, training accuracy 0.98
step 4200, training accuracy 1
step 4250, training accuracy 0.98
step 4300, training accuracy 0.99
step 4350, training accuracy 1
step 4400, training accuracy 1
step 4450, training accuracy 1
step 4500, training accuracy 0.99
step 4550, training accuracy 0.98
step 4600, training accuracy 0.99
step 4650, training accuracy 1
step 4700, training accuracy 0.99
step 4750, training accuracy 1
step 4800, training accuracy 0.98
step 4850, training accuracy 0.96
step 4900, training accuracy 1
step 4950, training accuracy 1
step 5000, training accuracy 1
step 5050, training accuracy 1
step 5100, training accuracy 1
step 5150, training accuracy 1
step 5200, training accuracy 0.99
step 5250, training accuracy 1
step 5300, training accuracy 1
step 5350, training accuracy 0.99
step 5400, training accuracy 0.99
step 5450, training accuracy 1
step 5500, training accuracy 0.99
step 5550, training accuracy 1
step 5600, training accuracy 0.99
step 5650, training accuracy 0.98
step 5700, training accuracy 1
step 5750, training accuracy 0.99
step 5800, training accuracy 1
step 5850, training accuracy 1
step 5900, training accuracy 0.99
step 5950, training accuracy 0.97
step 6000, training accuracy 0.99
step 6050, training accuracy 1
step 6100, training accuracy 0.99
step 6150, training accuracy 0.99
step 6200, training accuracy 0.98
step 6250, training accuracy 0.98
step 6300, training accuracy 1
step 6350, training accuracy 1
step 6400, training accuracy 1
step 6450, training accuracy 1
step 6500, training accuracy 0.99
step 6550, training accuracy 1
step 6600, training accuracy 1
step 6650, training accuracy 1
step 6700, training accuracy 0.99
step 6750, training accuracy 1
step 6800, training accuracy 1
step 6850, training accuracy 1
step 6900, training accuracy 1
step 6950, training accuracy 0.98
step 7000, training accuracy 0.98
step 7050, training accuracy 1
step 7100, training accuracy 0.99
step 7150, training accuracy 1
step 7200, training accuracy 1
step 7250, training accuracy 0.99
step 7300, training accuracy 1
step 7350, training accuracy 1
step 7400, training accuracy 0.99
step 7450, training accuracy 1
step 7500, training accuracy 0.99
step 7550, training accuracy 1
step 7600, training accuracy 1
step 7650, training accuracy 0.99
step 7700, training accuracy 1
step 7750, training accuracy 1
step 7800, training accuracy 1
step 7850, training accuracy 0.99
step 7900, training accuracy 0.99
step 7950, training accuracy 0.98
step 8000, training accuracy 1
step 8050, training accuracy 0.99
step 8100, training accuracy 1
step 8150, training accuracy 1
step 8200, training accuracy 0.99
step 8250, training accuracy 1
step 8300, training accuracy 1
step 8350, training accuracy 1
step 8400, training accuracy 1
step 8450, training accuracy 0.98
step 8500, training accuracy 0.99
step 8550, training accuracy 0.99
step 8600, training accuracy 1
step 8650, training accuracy 1
step 8700, training accuracy 1
step 8750, training accuracy 1
step 8800, training accuracy 0.98
step 8850, training accuracy 1
step 8900, training accuracy 1
step 8950, training accuracy 1
step 9000, training accuracy 1
step 9050, training accuracy 1
step 9100, training accuracy 1
step 9150, training accuracy 1
step 9200, training accuracy 1
step 9250, training accuracy 1
step 9300, training accuracy 1
step 9350, training accuracy 0.99
step 9400, training accuracy 1
step 9450, training accuracy 1
step 9500, training accuracy 1
step 9550, training accuracy 1
step 9600, training accuracy 1
step 9650, training accuracy 0.99
step 9700, training accuracy 1
step 9750, training accuracy 0.99
step 9800, training accuracy 1
step 9850, training accuracy 1
step 9900, training accuracy 0.99
step 9950, training accuracy 1
step 10000, training accuracy 1
step 10050, training accuracy 1
step 10100, training accuracy 1
step 10150, training accuracy 1
step 10200, training accuracy 1
step 10250, training accuracy 1
step 10300, training accuracy 0.99
step 10350, training accuracy 1
step 10400, training accuracy 1
step 10450, training accuracy 1
step 10500, training accuracy 0.99
step 10550, training accuracy 1
step 10600, training accuracy 1
step 10650, training accuracy 1
step 10700, training accuracy 1
step 10750, training accuracy 1
step 10800, training accuracy 1
step 10850, training accuracy 1
step 10900, training accuracy 1
step 10950, training accuracy 1
step 11000, training accuracy 1
step 11050, training accuracy 1
step 11100, training accuracy 1
step 11150, training accuracy 1
step 11200, training accuracy 1
step 11250, training accuracy 1
step 11300, training accuracy 1
step 11350, training accuracy 1
step 11400, training accuracy 1
step 11450, training accuracy 1
step 11500, training accuracy 1
step 11550, training accuracy 1
step 11600, training accuracy 1
step 11650, training accuracy 1
step 11700, training accuracy 1
step 11750, training accuracy 1
step 11800, training accuracy 1
step 11850, training accuracy 1
step 11900, training accuracy 1
step 11950, training accuracy 1
step 12000, training accuracy 0.99
step 12050, training accuracy 1
step 12100, training accuracy 1
step 12150, training accuracy 1
step 12200, training accuracy 1
step 12250, training accuracy 1
step 12300, training accuracy 1
step 12350, training accuracy 1
step 12400, training accuracy 1
step 12450, training accuracy 1
step 12500, training accuracy 1
step 12550, training accuracy 1
step 12600, training accuracy 1
step 12650, training accuracy 1
step 12700, training accuracy 1
step 12750, training accuracy 1
step 12800, training accuracy 1
step 12850, training accuracy 0.99
step 12900, training accuracy 1
step 12950, training accuracy 0.99
step 13000, training accuracy 1
step 13050, training accuracy 1
step 13100, training accuracy 1
step 13150, training accuracy 1
step 13200, training accuracy 1
step 13250, training accuracy 1
step 13300, training accuracy 1
step 13350, training accuracy 1
step 13400, training accuracy 1
step 13450, training accuracy 1
step 13500, training accuracy 1
step 13550, training accuracy 1
step 13600, training accuracy 1
step 13650, training accuracy 1
step 13700, training accuracy 1
step 13750, training accuracy 1
step 13800, training accuracy 1
step 13850, training accuracy 1
step 13900, training accuracy 1
step 13950, training accuracy 1
step 14000, training accuracy 1
step 14050, training accuracy 1
step 14100, training accuracy 1
step 14150, training accuracy 1
step 14200, training accuracy 1
step 14250, training accuracy 1
step 14300, training accuracy 1
step 14350, training accuracy 1
step 14400, training accuracy 1
step 14450, training accuracy 1
step 14500, training accuracy 1
step 14550, training accuracy 1
step 14600, training accuracy 1
step 14650, training accuracy 1
step 14700, training accuracy 1
step 14750, training accuracy 1
step 14800, training accuracy 1
step 14850, training accuracy 1
step 14900, training accuracy 1
step 14950, training accuracy 1
step 15000, training accuracy 1
step 15050, training accuracy 1
step 15100, training accuracy 1
step 15150, training accuracy 1
step 15200, training accuracy 1
step 15250, training accuracy 1
step 15300, training accuracy 1
step 15350, training accuracy 1
step 15400, training accuracy 1
step 15450, training accuracy 1
step 15500, training accuracy 1
step 15550, training accuracy 1
step 15600, training accuracy 1
step 15650, training accuracy 1
step 15700, training accuracy 1
step 15750, training accuracy 1
step 15800, training accuracy 1
step 15850, training accuracy 1
step 15900, training accuracy 1
step 15950, training accuracy 1
step 16000, training accuracy 1
step 16050, training accuracy 1
step 16100, training accuracy 1
step 16150, training accuracy 1
step 16200, training accuracy 1
step 16250, training accuracy 1
step 16300, training accuracy 1
step 16350, training accuracy 1
step 16400, training accuracy 1
step 16450, training accuracy 1
step 16500, training accuracy 1
step 16550, training accuracy 1
step 16600, training accuracy 1
step 16650, training accuracy 1
step 16700, training accuracy 1
step 16750, training accuracy 1
step 16800, training accuracy 1
step 16850, training accuracy 1
step 16900, training accuracy 1
step 16950, training accuracy 1
step 17000, training accuracy 1
step 17050, training accuracy 1
step 17100, training accuracy 1
step 17150, training accuracy 1
step 17200, training accuracy 1
step 17250, training accuracy 1
step 17300, training accuracy 1
step 17350, training accuracy 1
step 17400, training accuracy 1
step 17450, training accuracy 1
step 17500, training accuracy 1
step 17550, training accuracy 1
step 17600, training accuracy 1
step 17650, training accuracy 1
step 17700, training accuracy 1
step 17750, training accuracy 1
step 17800, training accuracy 1
step 17850, training accuracy 1
step 17900, training accuracy 0.99
step 17950, training accuracy 1
step 18000, training accuracy 1
step 18050, training accuracy 1
step 18100, training accuracy 1
step 18150, training accuracy 1
step 18200, training accuracy 1
step 18250, training accuracy 1
step 18300, training accuracy 1
step 18350, training accuracy 1
step 18400, training accuracy 1
step 18450, training accuracy 1
step 18500, training accuracy 1
step 18550, training accuracy 1
step 18600, training accuracy 1
step 18650, training accuracy 1
step 18700, training accuracy 0.99
step 18750, training accuracy 1
step 18800, training accuracy 0.99
step 18850, training accuracy 1
step 18900, training accuracy 1
step 18950, training accuracy 1
step 19000, training accuracy 1
step 19050, training accuracy 1
step 19100, training accuracy 1
step 19150, training accuracy 1
step 19200, training accuracy 1
step 19250, training accuracy 1
step 19300, training accuracy 1
step 19350, training accuracy 1
step 19400, training accuracy 0.99
step 19450, training accuracy 1
step 19500, training accuracy 1
step 19550, training accuracy 1
step 19600, training accuracy 1
step 19650, training accuracy 1
step 19700, training accuracy 1
step 19750, training accuracy 1
step 19800, training accuracy 1
step 19850, training accuracy 1
step 19900, training accuracy 1
step 19950, training accuracy 1
test accuracy 0.9928
>>> 
